x-pack/plugins/security_solution/server/endpoint/services/actions/utils.ts

```diff
@@ -431,3 +433,26 @@ export const formatEndpointActionResults = (
       })
     : [];
 };
+
+export const getAgentHostNamesWithIds = async ({
+  esClient,
+  searchedAgentIds,
+  metadataService,
+}: {
+  esClient: ElasticsearchClient;
+  searchedAgentIds: string[];
```

Comment:
(Optional): Suggestion: rename  to `agentIds`

"searcedAgentIds" does not make sense to me here in this utility